home_3_2.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"unicode/utf8"
+)
+
+func main() {
+
+	sliceWords := []string{"one", "thr22", "two", "three3333", "thr33"}
+	fmt.Println(max(sliceWords))
+
+	sliceNum := []int64{0, 1, 2, 4, 5, 67, 8}
+	fmt.Println(reverse(sliceNum))
+
+}
+
+func max(slice []string) string {
+	lenSrt := 0
+	var elem string
+	for _, v := range slice {
+
+		lenSrt = utf8.RuneCountInString(v)
+
+		if len(elem) < lenSrt {
```

Comment:
Please use `utf8.RuneCountInString()` to get length of `elem`

Reply:
It's ok to use ```len``` for ASCI chars but you should be consistent use one approach in function, I see for some part ```utf8.RuneCountInString``` used and ```len``` for  some part